pkg/gallery/models.go

```diff
@@ -110,9 +111,16 @@ func InstallModel(basePath, nameOverride string, config *Config, configOverrides
 		if err := utils.VerifyPath(file.Filename, basePath); err != nil {
 			return err
 		}
+
 		// Create file path
 		filePath := filepath.Join(basePath, file.Filename)
 
+		scanResults, err := downloader.HuggingFaceScan(file.URI)
```

Comment:
I'm a bit against having this by default - because every model installation would hit the HF endpoint and depend on its result. 

This fails shortly in an airgap environment, while models can be served with other means, not necessarly exposed to the internet. I know that we fail only in case we receive a response - but we hit everytime the HF endpoint in this case.

Reply:
I specifically rigged this to fail early and _not_ hit that external URL in the case that the endpoint isn't HF - that's the reason a lot of the Err checks have the slightly chunky `if err != nil && !errors.Is(err, downloader.ErrNonHuggingFaceFile) {` form. 

My logic here was that if the user's machine can't hit the `/scan` endpoint... they are likely going to have a hard time downloading the model itself just below